target/linux/generic/config-6.12

```diff
@@ -4166,6 +4238,7 @@ CONFIG_NEW_LEDS=y
 # CONFIG_NFC is not set
 # CONFIG_NFP is not set
 # CONFIG_NFSD is not set
+CONFIG_NFSD_LEGACY_CLIENT_TRACKING=y
```

Comment:
Maybe disable it? It's deprecated.
https://elixir.bootlin.com/linux/v6.12.19/source/fs/nfsd/Kconfig#L164

Reply:
Ok, I'll disable this in the next push.